client/src/components/CodeEditor.tsx

```diff
@@ -22,16 +22,13 @@ const CodeEditor = ({ language, code, readOnly }: Props) => {
   return (
     <Box
       sx={{
-        backgroundColor: "#282A36",
+        backgroundColor: "editor.background",
         width: "inherit",
         height: '100%',
         border: 1,
         borderColor: "divider",
         borderRadius: "12px",
         justifyItems: "flex-end",
-        marginLeft: 2,
-        paddingTop: 2,
-        paddingBottom: 2,
```

Comment:
these need to be re-added since they are needed for the code editor to have rounded corners, as in the design